src/com/seafile/seadroid2/ui/activity/AccountsActivity.java

```diff
@@ -235,9 +235,8 @@ public void onBackPressed() {
         Account account = accountManager.getCurrentAccount();
         if (account == null) {
             // force exit when current account was deleted
-            Intent i = new Intent();
-            i.setAction(Intent.ACTION_MAIN);
-            i.addCategory(Intent.CATEGORY_HOME);
+            Intent i = new Intent(this, BrowserActivity.class);
+            i.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
```

Comment:
Yes, you're right. But as you've already merged the alternative patch, I'm just closing this PR.